packages/lib/modules/tokens/TokensProvider.tsx

```diff
@@ -179,3 +180,15 @@ export function TokensProvider({
 }
 
 export const useTokens = (): UseTokensResult => useMandatoryContext(TokensContext, 'Tokens')
+
+export function useVebalToken() {
+  const { getTokensByChain } = useTokens()
+
+  const tokens = getTokensByChain(GqlChain.Mainnet)
+
+  const vebalBptToken = tokens.find(
+    t => t.address === mainnetNetworkConfig.tokens.addresses.veBalBpt
+  )
+
+  return vebalBptToken
+}
```

Comment:
Don't create a new hook for this. Just export the vebalBptToken const from the
tokens provider.